tests/visualization/times.json

```diff
@@ -0,0 +1 @@
+{"time": [0, 1], "dt": [1.0, 1.0]}
```

Comment:
Please purge file

Reply:
Done,
We might wanna think about adding `tests/*.json` to `.gitignore`.